buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1314,63 +1314,94 @@ artifactId=${project.name}
 
     /** ***********************************************************************************************/
 
+    // We specifically export the shadow closure and the dependencies closure so that they
+    // can be re-used by the beam-vendor-java-grpc-v1 project and within the applyPortabilityNature
+    // block.
+    project.ext.GRPC_V1_SHADOW_CLOSURE = {
```

Comment:
If this is used in just one place ever, can it be inlined?

Reply:
It is used in two places.

Reply:
Ah, I see where & why. Thanks!